lib/chef/provider.rb

```diff
@@ -376,7 +376,18 @@ module ClassMethods
         # compile the resources, converging them, and then checking if any
         # were updated (and updating new-resource if so)
         def action(name, &block)
-          define_method("action_#{name}") { compile_and_converge_action(&block) }
```

Comment:
@lamont-granquist It can be pulled in, but will not fix the `super` problem. The *only* way for `super` with no methods to work inside the `action :create do ... end` block is for it to be the top level of a defined method. We can remove the `super`-with-no-arguments capability but it will break some cookbooks based on 12.5.